### PR TITLE
Updated Student Workshop Docs to add CloudShell and GitBash instructions

### PR DESCRIPTION
**Pull request by manojsingh**
This PR includes documentation changes for the student workshop. The ReadMe file has been updated to reflect CloudShell and Git Bash instructions as well.

### CODE REVIEW

File: workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
-## Launch Github Codspaces
-This workshop leverages Github Codespaces to provide a development environment for running the instructions. Certainly you can run these instruction from a shell. But for people trying to get familiar with ASA-E for the first time, to rule out any environment related issues we recommend using Github Codespaces.
+#### There are multiple options to work through this workshop.
+- GitHub Codespaces
```
Comment (selvasingh) [inline workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: Please morph these bullets into hyperlinks to sections in README

File: workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md [hunk 2]
```diff
@@ -14,16 +21,63 @@ This workshop leverages Github Codespaces to provide a development environment f
...
+![Alt text](../../../../media/bashshell.png?raw=true "Optional Title")
+
+## Git Bash
+
+1. Install Visual Studio code: [Download Visual Studio Code - Mac, Linux, Windows](https://code.visualstudio.com/download)
+2. Install Azure CLI: [Install the Azure CLI for Windows | Microsoft Learn](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)
+3. Install GIT: [Git (git-scm.com)](https://git-scm.com/)
+4. Install jq utility: [Download jq (jqlang.github.io)](https://jqlang.github.io/jq/download/)
+ Please note that you will have to rename jq-win64.exe to jq.exe and add it to PATH.
+5. Install Java 17: [Red Hat build of OpenJDK Download | Red Hat Developer](https://developers.redhat.com/products/openjdk/download?sc_cid=701f2000000RWTnAAO)
```
Comment (selvasingh) [inline workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: We must use Microsoft Build of OpenJDK
https://www.microsoft.com/openjdk

See https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-17

File: workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md [hunk 3]
```diff
@@ -14,16 +21,63 @@ This workshop leverages Github Codespaces to provide a development environment f
...
+6. The environment variable **JAVA_HOME** should be set to the path of the JDK installation. The directory specified by this path should have bin, jre and lib among its subdirectories. Further, ensure your **PATH** variable contains the directory `${JAVA_HOME}/bin`. To test, type which `javac` into bash shell ensure the resulting path points to a file inside `${JAVA_HOME}/bin`.
+7. Install maven: [Maven – Installing Apache Maven](https://maven.apache.org/install.html)
```
Comment (selvasingh) [inline workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md:43]: Is this needed for this workshop?

**Review by selvasingh — CHANGES_REQUESTED** [review]

File: workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md (hunk 3 above)
Comment (manojsingh) [reply workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md:43]: This is needed only when using Git Bash for this workshop

**Review by selvasingh — APPROVED** [review]

**Pull request merged**